openvdb_ax/openvdb_ax/test/integration/TestStandardFunctions.cc

```diff
@@ -173,7 +48,7 @@ inline void testFunctionOptions(unittest_util::AXTestHarness& harness,
     const ast::Tree::Ptr syntaxTree = ast::parse(code.c_str());
     timer.stop();
 
-    CPPUNIT_ASSERT_MESSAGE(syntaxTree, "Invalid AX passed to testFunctionOptions.");
+    ASSERT_TRUE(syntaxTree) << "Invalid AX passed to testFunctionOptions.";
```

Comment:
The arguments to CPPUNIT_ASSERT_MESSAGE were reversed here. If I understand correctly then, this was ensuring that the string literal evaluates to true (which as a non-null pointer, it does) and would otherwise have printed  out the pointer to the AST (perhaps via shared_ptr's operator<<(ostream&))